imports/ui/unit/unit.jsx

```diff
@@ -161,20 +161,20 @@ class Unit extends Component {
       <MenuItem
         key={name}
         insetChildren
-        checked={statusFilterValues && statusFilterValues === {name}}
+        checked={statusFilterValues && statusFilterValues.indexOf(name) > -1}
```

Comment:
indexOf(name) > -1 might be a part of material UI implementation. The tick mark appears with this. 
[Material UI multiple selection example ](https://v0.material-ui.com/#/components/select-field)
<img width="505" alt="screen shot 2018-09-13 at 10 04 37 am" src="https://user-images.githubusercontent.com/25441733/45471667-62fa8d00-b764-11e8-935d-a45a0caf3bdd.png">

Reply:
I'm guessing that's in response to one of the previous comments I made. What I probably thought is that it's a string and not an array (both have the `indexOf` method). I was wrong, and that does make sense.